sdk/core/azure-core-http-netty/CHANGELOG.md

```diff
@@ -3,7 +3,11 @@
 ## 1.12.0-beta.1 (Unreleased)
 
 ### Features Added
-
+- Added a new configuration environment variable, `AZURE_DISABLE_HTTP_CLIENT_SHARING`, to support singleton `HttpClient`
+  instantiation. Now, `HttpClient` is a singleton instance by default. When the configuration variable sets to `true`,
+  it falls back to previous implementation which creates a new `HttpClient` instance when calling `createInstace()`
+  method of `NettyAsyncHttpClientProvider` everytime.
+  
 ### Breaking Changes
 
```

Comment:
We should probably move this to `Breaking Changes` section.

Also consider restructuring this entry in such a way that we describe what happened and then talk about compat switch. For example:

The `NettyAsyncHttpClientProvider.createInstance()` now returns shared instance of `HttpClient`. Azure libraries that don't configure custom `HttpClientOptions` or `HttpClient` instance are now using single shared instance as well. A new configuration environment variable  `AZURE_DISABLE_HTTP_CLIENT_SHARING` has been added for backwards compatibility setting it to `true` brings back previous behavior.